sdk_entrepot_gpf/workflow/action/ProcessingExecutionAction.py

```diff
@@ -232,17 +250,41 @@ def __create_processing_execution(self, datastore: Optional[str] = None) -> None
 
     def __add_tags(self) -> None:
         """Ajout des tags sur l'Upload ou la StoredData en sortie du ProcessingExecution."""
-        if "tags" not in self.definition_dict or self.definition_dict["tags"] == {} or self.__no_output:
-            # cas on a pas de tag ou vide: on ne fait rien
+        d_tags = self.definition_dict.get("tags", {})
+        # gestion des tags pour compatibility_cartes
+        if self.__mode_cartes and self.__processing_execution:
+            # mise en base de donnée livrée (vecteur)
+            if self.__processing_execution.id == Config().get_str("compatibility_cartes", "id_mise_en_base"):
+                if "datasheet_name" not in d_tags:
+                    raise GpfSdkError("Mode compatibility_cartes activé, il faut obligatoirement définir le tag 'datasheet_name'")
+                if not self.__inputs_upload or not self.stored_data:
+                    raise GpfSdkError("Intégration de données vecteur livrées en base : input and output obligatoires")
+                for o_upload in self.__inputs_upload:
+                    # ajout des tags génériques
+                    o_upload.api_add_tags({"proc_int_id": self.__processing_execution.id, "vectordb_id": self.stored_data.id})
+                    # ajout des tags spécifiques au mode cartes
+                    UploadAction.add_carte_tags(self.__mode_cartes, o_upload, "execution_start")
```

Comment:
@Alain2023 par rapport aux commentaires : dans les deux cas on ajoute des tags spécifiques au mode carte (cf. le if l. 255).
Les tags `proc_int_id` et `vectoredb_id` sont les tags spécifiques à ce traitement (identifiants de l'exec de traitements et de la donnée en sortie) et les autres tags sont spécifiques à l'étape (c'est donc toujours les mêmes, on les récupère en config grâce à la fonction `add_carte_tags`).

```suggestion
                    # ajout des tags "mode cartes" permettant d'identifier le traitement et la donnée stockée liés à la livraison
                    o_upload.api_add_tags({"proc_int_id": self.__processing_execution.id, "vectordb_id": self.stored_data.id})
                    # ajout des tags "mode cartes" permettant de suivre l'étape du traitement
                    UploadAction.add_carte_tags(self.__mode_cartes, o_upload, "execution_start")
```